src/GraphQL.SystemReactive/GraphQLBuilderExtensions.cs

```diff
@@ -0,0 +1,15 @@
+using GraphQL.DI;
+
+namespace GraphQL.SystemReactive
+{
+    /// <inheritdoc cref="GraphQL.GraphQLBuilderExtensions"/>
+    public static class GraphQLBuilderExtensions
+    {
+        /// <summary>
+        /// Registers <see cref="SubscriptionDocumentExecuter"/> as a singleton of type
+        /// <see cref="IDocumentExecuter"/> within the dependency injection framework.
+        /// </summary>
+        public static IGraphQLBuilder AddSubscriptionDocumentExecuter(this IGraphQLBuilder builder)
```

Comment:
What do you think about adding overload ?
```
 public static IGraphQLBuilder AddSubscriptionDocumentExecuter<TDocumentExecuter>(this IGraphQLBuilder builder) where TDocumentExecuter : SubscriptionDocumentExecuter
=>  builder.AddDocumentExecuter<TDocumentExecuter>();
```

Reply:
The same suggestion for other such methods.

Reply:
It seems excessive since AddDocumentExecuter already handles this scenario. Same is true for all 3rd party methods - we don’t need a special AddMemoryDocumentCache overload for classes that derive from MemoryDocumentCache for example. People use AddDocumentCache

Reply:
OK